src/pretalx/static/common/scss/_pretalx.scss

```diff
@@ -102,3 +102,9 @@
 .mail-card .card-header {
   cursor: pointer;
 }
+.nowline {
+    border-top: red solid 2px;
```

Comment:
It might be worth it to use `$brand-danger` here instead of `red`, allowing for easier styling.

Reply:
done